### ISSUE TITLE
Malformed zft files

### ISSUE THREAD

**Issue by kalwalt** [webarkit/Nft-Marker-Creator-App#18]
I tested the .zft files from the new version with jsartoolkitNFT and i got this error :

```
Error: 'fset' not found in the string.

_emscripten_console_error @ artoolkitNFT_wasm.simd.js:1
$func67 @ 001b8e22:0x2d33
$func1122 @ 001b8e22:0x5879e
$func1088 @ 001b8e22:0x50ecb
_decompressZFT @ VM46:6
loadZFT @ artoolkitNFT_wasm.simd.js:1
onSuccessZFT @ artoolkitNFT_wasm.simd.js:1
writeByteArrayToFS @ artoolkitNFT_wasm.simd.js:1
oReq.onload @ artoolkitNFT_wasm.simd.js:1
load
ajax @ artoolkitNFT_wasm.simd.js:1
addNFTMarkers @ artoolkitNFT_wasm.simd.js:1
ARControllerNFT.loadNFTMarkers @ artoolkitNFT_wasm.simd.js:1
ARControllerNFT.loadNFTMarker @ artoolkitNFT_wasm.simd.js:1
onLoad @ artoolkitNFT.worker.js:86
(anonymous) @ artoolkitNFT_wasm.simd.js:1
writeCallback @ artoolkitNFT_wasm.simd.js:1
writeByteArrayToFS @ artoolkitNFT_wasm.simd.js:1
oReq.onload @ artoolkitNFT_wasm.simd.js:1
load
ajax @ artoolkitNFT_wasm.simd.js:1
loadCamera @ artoolkitNFT_wasm.simd.js:1
ARCameraParamNFT.load @ artoolkitNFT_wasm.simd.js:1
ARCameraParamNFT @ artoolkitNFT_wasm.simd.js:1
(anonymous) @ artoolkitNFT.worker.js:114
scope.Module.onRuntimeInitialized @ artoolkitNFT_wasm.simd.js:1
doRun @ artoolkitNFT_wasm.simd.js:1
run @ artoolkitNFT_wasm.simd.js:1
runCaller @ artoolkitNFT_wasm.simd.js:1
removeRunDependency @ artoolkitNFT_wasm.simd.js:1
receiveInstance @ artoolkitNFT_wasm.simd.js:1
receiveInstantiationResult @ artoolkitNFT_wasm.simd.js:1
Promise.then
instantiateArrayBuffer @ artoolkitNFT_wasm.simd.js:1
instantiateAsync @ artoolkitNFT_wasm.simd.js:1
createWasm @ artoolkitNFT_wasm.simd.js:1
(anonymous) @ artoolkitNFT_wasm.simd.js:1
(anonymous) @ artoolkitNFT.worker.js:30
```
This means that the file not embedd the fset string inside, similar issue in PR https://github.com/webarkit/jsartoolkitNFT/pull/404 see comment https://github.com/webarkit/jsartoolkitNFT/pull/404#issuecomment-2491679529

i will try if previous versions are also affected.

**Comment by kalwalt** [comment]
This was present since version 0.1.0, this is very bad.... I'm looking for a solution, at least to find the source of the issue.

**Comment by kalwalt** [comment]
Instead .zft files generated by https://github.com/Carnaux/NFT-Marker-Creator works fine, just tested.
Anyone read this issue can generate his .zft files with it, notice that you need an old node version: 16.

**Comment by kalwalt** [comment]
working on this issue in PR #19

**Comment by kalwalt** [comment]
This was solved with #19, merged in the main branch.
